pages/common/nvim.md

```diff
@@ -2,6 +2,7 @@
 
 > Neovim, a programmer's text editor based on Vim, provides several modes for different kinds of text manipulation.
 > Pressing `i` in normal mode enters insert mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
+> See also `vim`, `vimtutor`,`vimdiff`.
```

Comment:
```suggestion
> See also `vim`, `vimtutor`, `vimdiff`.
```

- This change is optional - no corresponding style guide requirement exists.
- The same edit should be applied to other pages.
- It's better to be consistent in putting spaces here.